.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -90,6 +90,7 @@ on:
           - '["amdgpu"]'
           - '["Linux", "arc"]'
           - '["cts-cpu"]'
+          - '["Linux", "build"]'
```

Comment:
This should go to a separate PR so that we don't run needless CI jobs when this PR is updated.

Reply:
I think this only affects the options available for the manual e2e run. I dont think i see it affecting the pre-commit checks being ran. 
That being said do you know if there is a way to not run all the checks its doing due to being a sycl-devops-pr, and make it run the checks for a normal pr?